infra/modules/kubevela-addons/definitions/cardano-wallet-connector.cue

```diff
@@ -0,0 +1,171 @@
+parameter: {
+	network:       *"preview" | "preprod" | "mainnet"
+}
+
+#cardanoWalletConfigs: {
+  volumes: [{
+    name: "wallet-db"
+    persistentVolumeClaim: {
```

Comment:
If we're adding a `persistentVolumeClaim`, shouldn't we also add `k8s-update-strategy = Recreate`? If so, we should automatically do that and document it.

Reply:
done.